web/src/components/TimeSlider.tsx

```diff
@@ -74,12 +72,6 @@ export const getThumbIcon = (
   );
 };
 
-function trackTimeSliderEvent(selectedIndex: number, timeRange: TimeRange) {
-  trackEvent(TrackEvent.TIME_SLIDER_BUTTON, {
-    selectedIndex: `${timeRange}: ${selectedIndex}`,
-  });
-}
```

Comment:
We don't really use this and it's sending way to many requests when you move the slider to the point where it even gets rate limited!